features/api_services_list.feature

```diff
@@ -0,0 +1,235 @@
+              "name": "usg",
+              "status": null
+            },
+            {
+              "available": "yes",
+              "description": "Expanded Security Maintenance for Applications",
+              "entitled": null,
+              "name": "esm-apps",
+              "status": null
+            },
+            {
+              "available": "yes",
+              "description": "Expanded Security Maintenance for Infrastructure",
+              "entitled": null,
+              "name": "esm-infra",
+              "status": null
+            },
+            {
+              "available": "no",
+              "description": "NIST-certified FIPS crypto packages",
+              "entitled": null,
+              "name": "fips",
+              "status": null
+            },
+            {
+              "available": "no",
+              "description": "Preview of FIPS crypto packages undergoing certification with NIST",
+              "entitled": null,
+              "name": "fips-preview",
+              "status": null
+            },
+            {
+              "available": "no",
+              "description": "FIPS compliant crypto packages with stable security updates",
+              "entitled": null,
+              "name": "fips-updates",
+              "status": null
+            },
+            {
+              "available": "yes",
+              "description": "Management and administration tool for Ubuntu",
+              "entitled": null,
+              "name": "landscape",
+              "status": null
+            },
+            {
+              "available": "yes",
+              "description": "Canonical Livepatch service",
+              "entitled": null,
+              "name": "livepatch",
+              "status": null
+            },
+            {
+              "available": "yes",
+              "description": "Ubuntu kernel with PREEMPT_RT patches integrated",
+              "entitled": null,
+              "name": "realtime-kernel",
+              "status": null
+            },
+            {
+              "available": "no",
+              "description": "Security Updates for the Robot Operating System",
+              "entitled": null,
+              "name": "ros",
+              "status": null
+            },
+            {
+              "available": "no",
+              "description": "All Updates for the Robot Operating System",
+              "entitled": null,
+              "name": "ros-updates",
+              "status": null
+            }
+          ]
+        },
+        "meta": {
+          "environment_vars": []
+        },
+        "type": "ServicesList"
+      }
+      """
+
+    Examples: ubuntu release
+      | release | machine_type  |
+      | xenial  | lxd-container |
+      | bionic  | lxd-container |
+      | focal   | lxd-container |
+      | jammy   | lxd-container |
+      | noble   | lxd-container |
+
+  @uses.config.contract_token
+  Scenario Outline: Get services list when unattached
```

Comment:
s/unattached/attached

Reply:
Additionally, I think we can consolidate both tests, just for performance reasons